corehq/apps/app_manager/models.py

```diff
@@ -5078,6 +5080,10 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
     use_custom_suite = BooleanProperty(default=False)
     custom_base_url = StringProperty()
     cloudcare_enabled = BooleanProperty(default=False)
+
+    anonymous_cloudcare_enabled = BooleanProperty(default=False)
+    anonymous_cloudcare_hash = StringProperty(default=random_string)
```

Comment:
not yet, but it would be easy to add. this adds a layer of indirection to the app id. this'll now create urls like: `http://localhost:8000/a/anewproject/cloudcare/apps/home/L7VEUC/`